### PR TITLE
IAM permission instructions for `assume_role_arn` for RDS max policy size error

### PR DESCRIPTION
**Pull request by greedy52**
Related PR:
- https://github.com/gravitational/teleport/pull/31025

The previous PR documented how to use `assume_role_arn` but it is missing instructions on how to configure these IAM roles.

### CODE REVIEW

File: docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx [hunk 1]
```diff
@@ -78,6 +78,15 @@ You can reduce the policy size by separating them into multiple IAM roles. Use
   /labels/#auto-discovery)
   for more details.
   </Admonition>
+
+  Create or print the required IAM policies with the following commands and
+  attach them to the respective IAM roles:
+  - `teleport db configure aws create-iam --types redshift,redshift-serverless --name teleport-redshift-access`
+  - `teleport db configure aws print-iam --types redshift,redshift-serverless`
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx:85]:
````suggestion

```code
$ teleport db configure aws create-iam --types redshift,redshift-serverless --name teleport-redshift-access
$ teleport db configure aws print-iam --types redshift,redshift-serverless
```
````
Suggesting making this a `code` block for easier copy/pasting

File: docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx [hunk 2]
```diff
@@ -101,6 +110,14 @@ You can reduce the policy size by separating them into multiple IAM roles. Use
   ```
   The Database Service will use the IAM roles specified `assume_role_arn` for
   both discovery and authentication.
+
+  To bootstrap IAM permissions, run the bootstrap command for each `assume_role_arn`:
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx:114]:
```suggestion
  To bootstrap IAM permissions, run the bootstrap command for each `assume_role_arn`:
  
```
Adding an empty line before the `code` block for tidiness

File: docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx [hunk 3]
```diff
@@ -116,6 +133,14 @@ You can reduce the policy size by separating them into multiple IAM roles. Use
       aws:
           assume_role_arn: "arn:aws:iam::123456789012:role/example-rds-access-role"
   ```
+
+  To bootstrap IAM permissions, run the bootstrap command for each `assume_role_arn`:
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx:137]:
```suggestion
  To bootstrap IAM permissions, run the bootstrap command for each `assume_role_arn`:
  
```

File: docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx [hunk 4]
```diff
@@ -151,6 +176,14 @@ You can reduce the policy size by separating them into multiple IAM roles. Use
       - labels:
           "env": "prod"
   ```
+
+  Create or print the required IAM policies with the following commands and
+  attach them to the respective IAM roles:
+  - `teleport db configure aws create-iam --types rds --name teleport-rds-access`
+  - `teleport db configure aws print-iam --types rds`
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx:183]:
````suggestion

```code
$ teleport db configure aws create-iam --types rds --name teleport-rds-access
$ teleport db configure aws print-iam --types rds
```
````

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

**Review by smallinsky — APPROVED** [review]

**Pull request merged**